topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -707,13 +961,13 @@ Here, treatment is the primary factor that we are interested in. The sequencing
 
 > ### {% icon comment %} Comment
 >
-> We recommend that you add as many factors as you think may affect gene expression in your experiment. It can be the sequencing type like here, but it can also be the manipulation (if different persons are involved in the library preparation), other batch effects, etc...
+> We recommend that you add as many factors as you think which may affect gene expression in your experiment. It can be the sequencing type like here, but it can also be the manipulation (if different persons are involved in the library preparation), other batch effects, etc...
```

Comment:
```suggestion
> We recommend that you add all factors you think may affect gene expression in your experiment. It can be the sequencing type like here, but it can also be the manipulation (if different persons are involved in the library preparation), other batch effects, etc...
```